### PR TITLE
DOC: fix _validate_names docstring

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Review by gfyoung — CHANGES_REQUESTED** [review]
I get fixing the doc-string (because we aren't issuing a warning).

Not sure why we are removing the `return` part of the logic though.

**Comment by jbrockmendel** [comment]
Because the one place where the function is called, the returned value isn’t assigned to snything

File: pandas/io/parsers.py [hunk 1]
```diff
@@ -395,25 +395,17 @@ def _validate_integer(name, val, min_val=0):
 
 def _validate_names(names):
     """
-    Check if the `names` parameter contains duplicates.
-
-    If duplicates are found, we issue a warning before returning.
+    Raise ValueError if the `names` parameter contains duplicates.
```
Comment (jreback) [inline pandas/io/parsers.py]: can you add a Raises section

Comment (jbrockmendel) [reply pandas/io/parsers.py]: updated+green

**Comment by jreback** [comment]
thanks @jbrockmendel

**Pull request merged**